distribution/lib/Standard/Base/0.0.0-dev/src/Enso_Cloud/Internal/Existing_Enso_Asset.enso

```diff
@@ -100,36 +113,71 @@ type Existing_Enso_Asset
     from_json json -> Existing_Enso_Asset =
         title = get_required_field "title" json expected_type=Text
         id = get_required_field "id" json expected_type=Text
-        Existing_Enso_Asset.from_id_and_title id title
+        metadata = Asset_Metadata.from_json json
+        Existing_Enso_Asset.new_with_metadata id title metadata
+
+    ## PRIVATE
+    new_with_metadata id:Text title:Text metadata:Asset_Metadata -> Existing_Enso_Asset =
+        Existing_Enso_Asset.Value title id (asset_type_from_id id) metadata
```

Comment:
```suggestion
    ## PRIVATE
    new id:Text title:Text metadata:(Nothing|Asset_Metadata)=Nothing -> Existing_Enso_Asset =
        Existing_Enso_Asset.Value title id (asset_type_from_id id) metadata
```

Could we not have a single `new` here?

Reply:
I wanted to make it very clear when the asset is constructed with or without metadata. IMO it is a bit clearer when this is verbose. But I can merge them - initially there were more differences in the logic, now indeed it's very similar